File: C/380.c
#include <stdio.h>
#include <stdlib.h>

// 380 - ¡Me Caso! - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop;
    scanf("%d", &loop);
    while (loop != 0) {
        int i;
        int result = 0;
        for (i = 0; i<loop; i++) {
            int n = 0;
            scanf("%d", &n);
            result += n;
        }
        printf("%d\n", result);
        scanf("%d", &loop);
    }
    return 0;
}

File: Java/380.java
import java.util.Scanner;

/**
 *
 * 380 - ¡Me Caso! - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int gastos = sc.nextInt();
        while (gastos != 0) {
          int gastoFinal = 0;
            for (int i = 1; i <= gastos; i++) {
                int precio = sc.nextInt();
                gastoFinal += precio;
            }
            System.out.println(gastoFinal);
            gastos = sc.nextInt();
        }
    }

}
